From: Markus Elfring <elfring@users.sourceforge.net>
Subject: closures: use seq_putc() in debug_show()
Date: Sun, 14 Jul 2024 19:15:09 +0200

A single line break should be put into a sequence.  Thus use the
corresponding function "seq_putc".

This issue was transformed by using the Coccinelle software.

Link: https://lkml.kernel.org/r/e7faa2c4-9590-44b4-8669-69ef810277b1@web.de
Signed-off-by: Markus Elfring <elfring@users.sourceforge.net>
Cc: Kent Overstreet <kent.overstreet@linux.dev>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/closure.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/closure.c~closures-use-seq_putc-in-debug_show
+++ a/lib/closure.c
@@ -278,7 +278,7 @@ static int debug_show(struct seq_file *f
 			seq_printf(f, " W %pS\n",
 				   (void *) cl->waiting_on);
 
-		seq_puts(f, "\n");
+		seq_putc(f, '\n');
 	}
 
 	spin_unlock_irq(&closure_list_lock);
_
